payjoin/src/receive/v1/mod.rs

```diff
@@ -847,6 +844,42 @@ pub(crate) mod test {
         }
     }
 
+    #[test]
+    fn empty_candidates_inputs() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
I think we've made a pattern of using `BoxError` from `payjoin-test-utils`.
`pub type BoxError = Box<dyn std::error::Error + 'static>;`